packages/core/logger.ts

```diff
@@ -0,0 +1,3 @@
+import { getLogger } from "@dotkomonline/logger"
+
+export const logger = getLogger("core")
```

Comment:
This is fine for now, but in the future we want a specific logger in each service where needed, that way it's a lot easier to trace